rdtools/test/filtering_test.py

```diff
@@ -66,26 +65,20 @@ def setUp(self):
         # Note: Power is expected to be Series object because clip_filter makes
         #       use of the Series.quantile() method.
 
-    def test_tcell_filter(self):
+    def test_clip_filter_upper(self):
         filtered = clip_filter(self.power, quant=0.98,
-                               low_power_cutoff=0.1)
-
-        # Must not be all True, there should be some filtered values.
-        self.assertFalse(filtered.all())
+                               low_power_cutoff=0)
 
-        # Expect 99% of the 98th quantile to be cut off.
+        # Expect 99% of the 98th quantile to be filtered
         expected_result = self.power < (98 * 0.99)
         self.assertTrue((expected_result == filtered).all())
 
-    def test_tcell_filter_cutoff(self):
+    def test_tcell_filter_low_cutoff(self):
```

Comment:
This test simultaneously checks both the upper and lower filtering. If something breaks the upper side filtering both the upper and lower tests will fail, and this may confuse the developer trying to diagnose the problem. I suggest either implementing my suggestion to only check the first ~5 elements of the expected result, or add detailed comments to help developers interpret the test results.